rs/bitcoin/ckbtc/minter/src/state.rs

```diff
@@ -376,10 +376,8 @@ pub struct CkBtcMinterState {
     pub finalized_utxos: BTreeMap<Principal, BTreeSet<Utxo>>,
 
     /// Process one timer event at a time.
-    #[serde(skip)]
     pub is_timer_running: bool,
 
-    #[serde(skip)]
```

Comment:
Why are you removing the skips?

Reply:
The minter state `CkBtcMinterState` had the annotations`Serialize, serde::Deserialize`, which I removed. This is because one should not need to serialize/deserialize the state, since the state is re-created by replaying events, so those annotations did not make any sense. `#[serde(skip)]` is only useful on a field if the struct is annoted with `Serialize` or `Deserialize` (it's a compilation error otherwise)